src/PlusCalibration/CMakeLists.txt

```diff
@@ -155,21 +161,22 @@ IF(PLUSBUILD_BUILD_PlusLib_TOOLS)
       ConvertXmlToFcsv
       TemporalCalibration
       ProbeCalibration
-    DESTINATION bin
+    EXPORT PlusLib
+    DESTINATION "${PLUSLIB_BINARY_INSTALL}"
     COMPONENT RuntimeExecutables
     )
 ENDIF()
 
 # --------------------------------------------------------------------------
 # Install
 #
-INSTALL(TARGETS vtkPlusCalibration
-  RUNTIME DESTINATION bin CONFIGURATIONS Release COMPONENT RuntimeLibraries
-  LIBRARY DESTINATION lib CONFIGURATIONS Release COMPONENT RuntimeLibraries
-  ARCHIVE DESTINATION lib CONFIGURATIONS Release COMPONENT Development
+INSTALL(TARGETS vtkPlusCalibration EXPORT PlusLib
+  RUNTIME DESTINATION "${PLUSLIB_BINARY_INSTALL}" CONFIGURATIONS Release COMPONENT RuntimeLibraries
+  LIBRARY DESTINATION "${PLUSLIB_LIBRARY_INSTALL}" CONFIGURATIONS Release COMPONENT RuntimeLibraries
+  ARCHIVE DESTINATION "${PLUSLIB_LIBRARY_INSTALL}" CONFIGURATIONS Release COMPONENT Development

```

Comment:
Could you add PLUSLIB_ARCHIVE_INSTALL, in case we want to change the location in the future.